schemas/session-replay/text-wireframe-schema.json

```diff
@@ -0,0 +1,133 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "text-wireframe-schema.json",
+  "title": "TextWireframe",
+  "type": "object",
+  "description": "Schema of all properties of a TextWireframe.",
+  "allOf": [
+    {
+      "$ref": "_common-shape-wireframe-schema.json"
+    },
+    {
+      "required": [
+        "text"
+      ],
+      "properties": {
+        "type": {
+          "type": "string",
+          "description": "The type of the wireframe.",
+          "const": "text",
+          "readOnly": true
+        },
+        "text": {
+          "type": "string",
+          "description": "The text value of the wireframe.",
+          "readOnly": false
+        },
+        "textStyle": {
+          "type": "object",
+          "description": "The style of this text field.",
+          "required": [
+            "family",
+            "font",
+            "color"
+          ],
+          "properties": {
```

Comment:
There was a good idea (i.a. from @louiszawadzki) to also include font type property which will help with finding a backup font (e.g. "serif" or "sans-serif").

Reply:
> Great work 💪, definitely a good start. I left smaller comments - mainly on optionality of fields vs their defaults.
> 
> Overal, I wonder on two things:
> 
> * Shouldn't we call this format (the folder) a `mobile-session-replay` instead of `session-replay`? Both mean something different.
> * Should we merge it directly to the `master` branch? I think it's fair to assume that at the beginning there will be a lot of iterations and breaking changes so maybe it's worth using feature branch for a while. On the other hand, both formats are isolated, so I'm ok with any approach.

@ncreated 
1. Regarding the format name I would rather keep it `session-replay` as this might relate with the browser session replay later. @BenoitZugmeyer What do you think ?
2. Like you also said, they are separated so I would rather merge them on master but I am waiting for others opinion also.

Reply:
> Shouldn't we call this format (the folder) a mobile-session-replay instead of session-replay? Both mean something different.

I think the directory can be called `session-replay` but then the `session-replay-events-schema.json` should reference mobile, e.g. `session-replay-events-mobile.json`. This makes room for browser ones to be added later. Wireframe definitions can keep their current names, there's little chance for collision with browser.

Reply:
I would not worry to much about that because in the end, the entry point is going to be segment with both browser & mobile datastructure

Reply:
To be a bit more precise, the way I see it when we documented everything, it would look something like that:
```
schema/session-replay
schema/session-replay/segment.json
schema/session-replay/record.json
schema/session-replay/meta-record.json
schema/session-replay/focus-record.json
schema/session-replay/viewport-record.json
schema/session-replay/mobile/fullsnapshot-record.json
schema/session-replay/mobile/shape-wireframe.json
schema/session-replay/mobile/text-wireframe.json
schema/session-replay/browser/fullsnapshot-record.json
schema/session-replay/browser/serialized-node.json
```
What do you think?
(this is out of scope of the current PR)

Reply:
Exactly what I was thinking also. If we all agree I can refactor to this folders architecture in the next PR where I am going to add the segment, records schemas.

Reply:
I like what @ThibautGeriz is suggesting. We'll consider adding browser-related schemas in the future.

Reply:
We'll have to it if we want to use the typescript type generated by the this repo.

Reply:
@ThibautGeriz can't you use both the Browser SDK types and the types from this repo at first?

Reply:
@BenoitZugmeyer Kinda but it's gonna be awkward, we'll start with that. As I target I wish we could avoid. The segment we fetch we'll have to cast it `MobileSegment | BrowserSegment` with both having the same definition for the focus record for example